webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -42,6 +42,8 @@ import org.knora.webapi.responders.IriService
 import org.knora.webapi.responders.Responder
 import org.knora.webapi.responders.admin.PermissionsResponder
 import org.knora.webapi.responders.v2.resources.CreateResourceV2Handler
+import org.knora.webapi.slice.admin.api.model.Project
+import org.knora.webapi.slice.admin.domain.model.KnoraProject
```

Comment:
Have imports been added but not used? I guess fatal warnings is not on. :(

Reply:
[rm unused imports](https://github.com/dasch-swiss/dsp-api/pull/3433/commits/29d56973db0bd4431299f10b07678eb24c442b1a)